### DISCUSSION TITLE
DA simulation interrupted

### DISCUSSION THREAD

**Discussion by Arijit22IITDelhi** [NASA-LIS/LISF#1473] [Q&A] [answered]
Hello, I was running SMAP DA from 2015 to 2022 but from 2020 onwards it is giving daily outputs at an time interval of 1 hour or more where initially it was 2 minutes or less. Now there is a chance of simulation getting stopped because of wall time limit (168 hours)
![image](https://github.com/NASA-LIS/LISF/assets/137404009/0a1dc13a-85f4-4696-b5e5-1d0946e14f72)
Please suggest me if the simulation stops in between  then how to start again from where it ends. From where should I take restart file and run the rest of simulations.

**Comment by dmocko** [comment, marked as answer]
HI @Arijit22IITDelhi 

What forcing are you using?  When using GDAS, we have noticed the runs can take significantly longer in later years, due to the larger file size (from finer spacial resolution) in the GDAS data.  It takes LIS longer to read these longer files, which will result in longer times between writing output files.

**Reply by Arijit22IITDelhi** [reply]
Thank you so much, the issue got resolved later.
